src/components/modals/Agenda/AgendaClassItem.tsx

```diff
@@ -65,7 +67,7 @@ export default function AgendaClassItem({
                     alignItems: "center",
                     justifyContent: "space-between",
                     background:
-                        agendaClass._class === undefined
+                        agendaClass._class === undefined || !bookingActive
```

Comment:
I think that using the same background for ghost and paused bookings could be confusing. I think the Alert might be sufficient, or maybe you could include the pause icon again in each AgendaClassItem.

Reply:
Yeah, should be more than enough to have the alert on top I think. Fixed in
https://github.com/mathiazom/rezervo-web/pull/100/commits/f04dd26a23c49797e596c7d83c0eb97926e0a1f2